main.go

```diff
@@ -14,27 +21,81 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if err = run(config); err != nil {
+		config.Logger.Error(errors.Wrap(err, "startup"))
+		os.Exit(1)
+	}
+}
+
+const shutdownTimeoutSeconds = 3
+
+func run(conf *sat.Config) error {
+	localLogger := conf.Logger.CreateScoped("main.run")
```

Comment:
Why do we need a scoped logger?

Reply:
we don't _need_ one, but it's nice to have one. In my ideal service, each part of a codebase would have their own scoped loggers, so each entry would not only have the log message attached to them, but also what scope (ie what part of the codebase it's coming from).